### ISSUE TITLE
FIX: Moving SceneModel does not update its AABB

### ISSUE THREAD

**Issue by hamza-hajji** [xeokit/xeokit-sdk#1275]
**Describe the bug**
Xeokit build: `2.4.2-beta-20`
after changing a model's `position`, CameraFlight#flyTo flies to original position

**To Reproduce**
1. Go to example https://xeokit.github.io/xeokit-sdk/examples/buildings/#xkt_vbo_Duplex
2. Move the model by running the script

`viewer.scene.models.myModel.position = [0, 20, 0]`

3. Fly to model using `model.aabb`, (I also tried using an object's aabb)

```js
viewer.cameraFlight.flyTo({
    aabb: viewer.scene.models.myModel.aabb,
});
```

The camera flies under the model

**Expected behavior**
Camera flies to the new model's position, this used to work in version `2.4.2-beta-7`

**Screenshots**

![image](https://github.com/xeokit/xeokit-sdk/assets/26790341/b5d4cfb2-d2a8-4150-91cd-e96916c7756d)

**Comment by hamza-hajji** [comment]
It turns out, the aabb of the model doesn't change

![image](https://github.com/xeokit/xeokit-sdk/assets/26790341/dc67131b-f983-42f2-96f2-907a24ab243a)

**Comment by xeolabs** [comment]
Now fixed in @xeokit/xeokit-sdk 2.4.2-beta-21
